packages/integration-sdk-cli/src/commands/collect.ts

```diff
@@ -30,15 +33,34 @@ export function collect() {
       collector,
       [],
     )
+    .option(
+      '-C, --use-dependencies-cache [filePath]',
+      'Loads cache for the dependencies required by the step(s) specified in --step option. Execution of these steps is skipped. Data found in .j1-integration is used if no filepath is provided.',
```

Comment:
More so for curiousity but we have the ability to create multiple dependency graphs. Have you tried to run that use case/does your code account for that? Maybe something we can address in a later PR if not.

Reply:
I have not. I'd love to see that in action!

Reply:
Opened #619 to track